decidim-debates/app/forms/decidim/debates/admin/close_debate_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    module Admin
+      # This class holds a Form to close debates from Decidim's admin views.
+      class CloseDebateForm < Decidim::Form
+        include TranslatableAttributes
+
+        mimic :debate
+
+        translatable_attribute :conclusions, String do |translated_attribute, locale|
+          validates translated_attribute, presence: true, if: ->(record) { record.default_locale?(locale) }
+          validates translated_attribute, length: { minimum: 10, maximum: 10_000 }, if: ->(record) { record.default_locale?(locale) }
```

Comment:
is there a reason to declare the validations into a block or it is optional?

Reply:
It's optional, but doing it this ways allows us to have introspection on the form builder and automatically detect the validations. This way we can add the length limits on the frontend.